test/common/tcp_proxy/tcp_proxy_test.cc

```diff
@@ -326,18 +336,86 @@ TEST(ConfigTest, DEPRECATED_FEATURE_TEST(Routes)) {
     connection.remote_address_ =
         std::make_shared<Network::Address::Ipv6Instance>("2003:0:0:0:0::5");
     EXPECT_EQ(std::string("with_v6_source_and_destination"),
-              config_obj.getRouteFromEntries(connection));
+              config_obj.getRouteFromEntries(connection)->clusterName());
   }
 
   {
     // fall through
     NiceMock<Network::MockConnection> connection;
     connection.local_address_ = std::make_shared<Network::Address::Ipv6Instance>("2004::");
     connection.remote_address_ = std::make_shared<Network::Address::Ipv6Instance>("::");
-    EXPECT_EQ(std::string("catch_all"), config_obj.getRouteFromEntries(connection));
+    EXPECT_EQ(std::string("catch_all"), config_obj.getRouteFromEntries(connection)->clusterName());
   }
 }
 
+TEST(ConfigTest, WeightedClusterWithZeroWeightConfig) {
```

Comment:
nit: Please add one-line comments describing the test (e.g. `// Tests invalid weighted cluster config`).

Reply:
done